tools/nni-auto-tune/main.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
```

Comment:
Seems that the license header is outdated, please refer to [here](https://github.com/smallv0221/SPTAG/blob/add-ann-auto-tune/setup.py)

Reply:
Done.